.github/workflows/continuous-deploy.yml

```
@@ -49,6 +49,12 @@ jobs:
           projectName: ubiquity-dollar
           directory: ./dist
 
+      - name: Nix CI Deploy
+        run: |
+          cd dist
+          RAILWAY_TOKEN=${{ secrets.RAILWAY_API_TOKEN }} yes | npx @railway/cli link ${{ secrets.RAILWAY_PROJECT_ID }}
+          RAILWAY_TOKEN=${{ secrets.RAILWAY_API_TOKEN }} yes | npx @railway/cli up
```

This is what I meant by exporting the environment variable.

```suggestion
          export RAILWAY_TOKEN=${{ secrets.RAILWAY_API_TOKEN }} yes | npx @railway/cli link ${{ secrets.RAILWAY_PROJECT_ID }}
          export RAILWAY_TOKEN=${{ secrets.RAILWAY_API_TOKEN }} yes | npx @railway/cli up
```